### PR TITLE
Add `tab` key code helper

### PR DESCRIPTION
**Pull request by cpjolicoeur**
Added the `tab` key as a helper key code and also formatted the files using `elm-format` for consistency.

### CODE REVIEW

File: tests/KeysTest.elm [hunk 1]
```diff
@@ -141,4 +141,20 @@ esc action =
...
+            Encode.object
+                [ ( "keyCode", int Keys.tab )
+                , ( "shiftKey", bool False )
+                , ( "metaKey", bool False )
+                , ( "ctrlKey", bool False )
+                , ( "altKey", bool False )
+                , ( "repeat", bool False )
+                ]
+    in
+    Event.custom action props
```
Comment (derrickreimer) [inline tests/KeysTest.elm]: It looks like this is not actually getting called in the tests. The other similarly named helper functions are used in the `eventTests` function to prove various behaviors of the event bindings. Probably better to remove this for now until a use for it arises.

**Comment by cpjolicoeur** [comment]
Yes, `elm-format 0.17.0.7.0-exp`

If you prefer, I can change to a non `exp` version?

File: tests/KeysTest.elm (hunk 1 above)
Comment (cpjolicoeur) [reply tests/KeysTest.elm]: Ahh yes, I'll add a usage to the `eventTests` if you prefer that?  At least it would have minimal test coverage?

**Comment by derrickreimer** [comment]
Let's keep it at `elm-format-0.18 0.6.1-alpha` for now 👍

File: tests/KeysTest.elm (hunk 1 above)
Comment (cpjolicoeur) [reply tests/KeysTest.elm]: NM, I'll just remove.  The addtion is so simple, there is not much value there to just test an additional `Int` keycode

**Comment by cpjolicoeur** [comment]
Updated with just the `tab` addition and used `elm-format` 0.6.1-alpha so those formatting changes were dropped completely.

**Comment by derrickreimer** [comment]
Sweet 💯

**Comment by derrickreimer** [comment]
Version 1.1.0 released to Elm packages.

**Pull request merged**